src/config/graph.rs

```diff
@@ -37,24 +37,28 @@ impl Graph {
         sources: &IndexMap<ComponentKey, SourceOuter>,
         transforms: &IndexMap<ComponentKey, TransformOuter<String>>,
         sinks: &IndexMap<ComponentKey, SinkOuter<String>>,
+        enrichment_tables: &IndexMap<ComponentKey, EnrichmentTableOuter<String>>,
```

Comment:
If you wanted to simplify somewhat, I think that these tables could be added to the graph as sinks. There's not really any function difference between them in this context.

Reply:
Yeah, that would probably makes things much easier to handle. Similar to how it was done in topology builder. I will make that update.